jquery.pickadate/jquery.pickadate.d.ts

```diff
@@ -1,69 +1,19 @@
-// Type definitions for pickadate.js 3.3.0
+// Type definitions for pickadate.js 3.5.5
 // Project: https://github.com/amsul/pickadate.js
 // Definitions by: Theodore Brown <https://github.com/theodorejb/>
 // Definitions: https://github.com/borisyankov/DefinitelyTyped
 
 ///<reference path="../jquery/jquery.d.ts" />
 
-/** Options shared between date and time pickers */
-interface pickerOptions {
-    /** Set clear button text */
-    clear?: string; // default 'Clear'
-
-    /** Specify where to insert the picker's root element by passing any valid CSS selector to this option */
-    container?: any;
-
-    // Events
-    onStart?: (event: any) => void;
-    onRender?: (event: any) => void;
-    onOpen?: (event: any) => void;
-    onClose?: (event: any) => void;
-    onSet?: (event: any) => void;
-    onStop?: (event: any) => void;
-}
-
-interface pickadateOptions extends pickerOptions {
-    // Strings and translations
-    monthsFull?: string[]; // default 'January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'
-    monthsShort?: string[]; // default 'Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'
-    weekdaysFull?: string[]; // default 'Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'
-    weekdaysShort?: string[]; // default 'Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'
-    showMonthsShort?: boolean;
-    showWeekdaysFull?: boolean;
-
-    // Buttons
-    today?: string; // default 'Today'
-
-    // Formats
-    format?: string; // default 'd mmmm, yyyy'
-    formatSubmit?: string; // e.g. 'yyyy/mm/dd'
-    hiddenPrefix?: string; // default undefined
-    hiddenSuffix?: string; // default '_submit'
-    hiddenName?: boolean;  // default undefined
-
-    // Dropdown selectors
-    selectYears?: any; // Specify the number of years selectable using an even integer - half before and half after the year in focus:
-    selectMonths?: boolean;
-
-    // First day of the week
-    firstDay?: any; // The first day of the week can be set to either Sunday or Monday. Anything truth-y sets it as Monday and anything false-y as Sunday
-
-    // Date limits
-    min?: any; // date object, array formatted as [YEAR,MONTH,DATE], or dates relative to today using integers or a boolean (`true` sets it to today. `false` removes any limits).
-    max?: any;
-
-    // Disable dates
-    disable?: any[]; // Date objects, arrays formatted as [YEAR,MONTH,DATE], or integers representing days of the week (from 1 to 7). Switch to whitelist by setting first item in collection to `true`.
-
-    // Classes
-    klass?: {
-
+declare module Pickadate {
+    /** KlassOptions shared between date and time pickers */
```

Comment:
looks weird to use JSDoc style comment blocks `/** */` for single line comments...